src/stim/simulators/frame_simulator.perf.cc

```diff
@@ -26,7 +26,7 @@ BENCHMARK(FrameSimulator_depolarize1_100Kqubits_1Ksamples_per1000) {
     size_t num_samples = 1000;
     double probability = 0.001;
     std::mt19937_64 rng(0);  // NOLINT(cert-msc51-cpp)
-    FrameSimulator sim(stats, FrameSimulatorMode::STORE_DETECTIONS_TO_MEMORY, num_samples, rng);
+    FrameSimulator<MAX_BITWORD_WIDTH> sim(stats, FrameSimulatorMode::STORE_DETECTIONS_TO_MEMORY, num_samples, rng);
```

Comment:
This is fine for now. I am definitely gonna do some sort of follow-up setting the width to various fixed numbers.